configs/components/runtime-pe-bolt-server.rb

```diff
@@ -8,6 +8,15 @@
     platform.provision_with "echo 'Acquire::AllowInsecureRepositories \"true\";' > /etc/apt/apt.conf.d/90insecure"
   end
 
-  platform.add_build_repository("http://enterprise.delivery.puppetlabs.net/#{settings[:pe_version]}/repos/#{platform.name}/#{platform.name}.repo")
+  artifactory_url = 'https://artifactory.delivery.puppetlabs.net/artifactory'
+
+  if platform.is_rpm?
+    platform.add_build_repository "#{artifactory_url}/rpm_enterprise__local/#{settings[:pe_version]}/repos/#{platform.name}/#{platform.name}.repo"
```

Comment:
It looks like this doesn't exist, at least for https://artifactory.delivery.puppetlabs.net/artifactory/rpm_enterprise__local/2019.2/repos/el-8-x86_64/ ...

Reply:
@puppetlabs/skeletor do we need to continue to build ofr 2019.2?

Reply:
Yeah i dont think we want to build that. I need to do some cleanup for 2019.2. here is a start, https://github.com/puppetlabs/ci-job-configs/pull/6841 but there is more.